content/how-to/dedicated-hosting/create-new-user-on-active-directory-domain/index.md

```diff
@@ -19,33 +19,36 @@ Active Directory user.
 1. Log in to the [MyRackspace portal](https://login.rackspace.com/login) with
    your username and password.
 
-2. In the top navigation bar, choose **Select a Product** > **Rackspace Dedicated**.
+   <img width="409" alt="newuser3" src="/support/how-to/create-new-user-on-active-directory-domain/newuser3.png">
 
-3. Select **Tickets** > **Create Ticket**. The **Create New Ticket** page displays.
+2. In the top navigation bar, choose **Supprt** > **Support Center**.
 
-4. Select the **Subject** field.
+   <img width="941" alt="newuser4" src="/support/how-to/create-new-user-on-active-directory-domain/newuser4.png">
 
-5. From the drop-down menu, select **Create New User on Active Directory Domain**.
+3. From the Common Request menu select **Create New User on Active domain directory** as shown in following image.
 
-   {{<image src="newuser1.png" alt="" title="">}}
+   <img width="811" alt="newuser5" src="/support/how-to/create-new-user-on-active-directory-domain/newuser5.png">
 
-6. Specify a username for the account you want to create.
+4. You can view **Issue Details** section by selecting **Create New User on Active domain directory** as shown in following image.
+
+   <img width="494" alt="newuser6" src="/support/how-to/create-new-user-on-active-directory-domain/newuser6.png">
+
+5. Specify a username for the account you want to create.
 
    After the request completes, the system generates a secure password automatically for
    this user and provides it in the ticket. You can change this password later.
 
-7. Specify a domain for the user.
+6. Specify a domain for the user.
 
-   Use the **Give this user Administrative or sudo permissions** checkbox to
-   grant the user administrative or sudo permissions. This automatically adds
+   Check box to grant the user administrative or sudo permissions. This automatically adds
    the user to Domain Administrator groups.
 
-   Use the **Additional Instructions** box to provide additional information,
+   Check box **Additional Instructions** to provide additional information,
    such as user details or group membership permissions.
 
-8. Select **Create Ticket**.
+7. Select **Submit**.
 
    After the ticket generates, you see your ticket list in the MyRackspace Portal.
    It typically takes a few minutes for the ticket creation process to complete.
 
-  {{<image src="newuser2.png" alt="" title="">}}
+  <img width="494" alt="newuser7" src="/support/how-to/create-new-user-on-active-directory-domain/newuser7.png">
```

Comment:
fix alt tag

Reply:
enter domain name and user name to create ticket